tests/StarCluster_AMR.in

```diff
@@ -0,0 +1,48 @@
+# *****************************************************************
+# Problem size and geometry
+# *****************************************************************
+geometry.prob_lo     =  -10.  -10.  -10.
+geometry.prob_hi     =  10.  10.  10.
+geometry.is_periodic =  0    0    0
+
+# *****************************************************************
+# VERBOSITY
+# *****************************************************************
+amr.v              = 1       # verbosity in Amr
+
+# *****************************************************************
+# Resolution and refinement
+# *****************************************************************
+amr.n_cell          = 512 512 512
+amr.max_level       = 4     # number of levels = max_level + 1
+amr.blocking_factor = 32    # grid size must be divisible by this
+amr.max_grid_size   = 128   # at least 128 for GPUs
+amr.n_error_buf     = 3     # minimum 3 cell buffer around tagged cells
+amr.grid_eff        = 0.7   # default
+
+hydro.reconstruction_order = 3  # PPM
+cfl = 0.2
+max_timesteps = 10000
```

Comment:
Are you sure it's running until `stop_time`? I strongly suspect it's hitting `max_timesteps` first.

Reply:
yes, its running until `stop_time`

```
Coarse STEP 6209 at t = 0.4999984763 (99.99969526%) starts ...
```